src/style/productCard.css

```diff
@@ -0,0 +1,85 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
```

Comment:
Is it necessary for this task?

Reply:
![image](https://user-images.githubusercontent.com/32616842/138078484-7c127f97-3204-4d62-b596-c18fc9912cdc.png)

Such a condition. And also I use align-self: center; for the picture